src/sentry/workflow_engine/processors/detector.py

```diff
@@ -55,6 +57,97 @@ def get_detector_by_event(event_data: WorkflowEventData) -> Detector:
     return detector
 
 
+def get_detectors_by_events_bulk(
```

Comment:
it'd be nice if we can decompose this method more -- for example, we could do ssomething like `extract_event_info` and pass the GroupEvent